src/ui/layout/mct-tree.vue

```diff
@@ -449,6 +454,53 @@ export default {
                 navigateToParent: navToParent
             };
         },
+        observeAncestors() {
+            let observedAncestorIds = Object.keys(this.observedAncestors);
+
+            // observe any ancestors, not currently being observed
+            this.ancestors.forEach((ancestor) => {
+                let ancestorObject = ancestor.object;
+                let ancestorKeyString = this.openmct.objects.makeKeyString(ancestorObject.identifier);
+                let index = observedAncestorIds.indexOf(ancestorKeyString);
+
+                if (index !== -1) { // currently observed
+                    observedAncestorIds.splice(index, 1); // remove all active ancestors from id tracking
+                } else { // not observed, observe it
+                    this.observeAncestor(ancestorKeyString, ancestorObject);
+                }
+            });
+
+            // remove any ancestors currnetly being observed that are no longer active ancestors
+            observedAncestorIds.forEach((id) => {
```

Comment:
there is a helper function called stopObservingAncestors, could we use that here instead of repeating code? You will probably have to move it above line 461, to make sure you are unobserving the correct ancestors.

Reply:
Ok, removed stopObservingAncstor (no 's') method, and moved it into stopObservingAndstors and set the default to be all of them, but now we can pass in a subset as we discussed.